VM/vm_params2vm.py

```diff
@@ -0,0 +1,288 @@
+#!/usr/bin/env python3
+"""
+REQUIREMENTS:
+PATH contains NZVM binary (github:ucgmsim/Velocity-Model/NZVM)
+
+Most basic example:
+
+mpirun -n 3 vm_params2vm.py Hossack ./vm_params.yml ~/Data/VMs
```

Comment:
This script should output the VM files directly into the directory given, instead of nesting one deeper. This is supposed to just be a script to make a VM from a parameter file. Anything related to the simulation directory structure will be handled by the orchestration script, or handled around this script.
Also this program is only MP (well NZVM is), not MPI, so shouldn't need mpirun?

Reply:
True - the comment was outdated